lib/moog-require.js

```diff
@@ -21,27 +20,28 @@ module.exports = function(options) {
   self.improvements = {};
 
   if (self.options.bundles) {
-    _.each(self.options.bundles, function(bundleName) {
+    for (const bundleName of Object.values(self.options.bundles)) {
       const bundlePath = getNpmPath(self.root.filename, bundleName);
       if (!bundlePath) {
         throw 'The configured bundle ' + bundleName + ' was not found in npm.';
       }
-      const imported = importFresh(bundlePath);
```

Comment:
I was nervous, but it looks like at this point in the code at least, `importFresh` was not important because all we're doing with `imported` is testing some properties and capturing the file path. OK so far.